src/lib/Radio.svelte

```diff
@@ -3,7 +3,7 @@
   import { ErrorMessage } from "./index.js";
 
   // A label for the entire group of radio buttons
-  export let legend: string;
+  export let label: string;
```

Comment:
When we update a repo, we'll have to change this. TS will make sure we do it right. I'm in favor of this change; I dunno why the names were inconsistent